src/test/java/com/google/sps/data/DatastoreScheduledInterviewTest.java

```diff
@@ -0,0 +1,185 @@
+            "user@company.org",
+            "user2@mail.com");
+
+    List<Boolean> actual = new ArrayList<Boolean>();
+    actual.add(copyScheduledInterview1.equals(result.get(0)));
+    actual.add(result.size() == 2);
+    actual.add(copyScheduledInterview2.equals(result.get(1)));
+
+    List<Boolean> expected = new ArrayList<Boolean>();
+    expected.add(true);
+    expected.add(true);
+    expected.add(true);
+
+    Assert.assertEquals(true, expected.equals(actual));
+  }
+
+  // Tests deleting a user's scheduledInterview.
+  @Test
+  public void deletesScheduledInterview() {
+    tester.create(scheduledInterview1);
+    tester.create(scheduledInterview2);
+    List<ScheduledInterview> result = tester.getForPerson("user@company.org");
+    tester.delete(result.get(0).id());
+    Entity entity = datastore.prepare(new Query("ScheduledInterview")).asSingleEntity();
+    ScheduledInterview storedScheduledInterview = tester.entityToScheduledInterview(entity);
+    ScheduledInterview copyScheduledInterview2 =
+        ScheduledInterview.create(
+            storedScheduledInterview.id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+            "user@company.org",
+            "user2@mail.com");
+    Assert.assertEquals(true, storedScheduledInterview.equals(copyScheduledInterview2));
+  }
+
+  // Tests updating a user's scheduledInterview.
+  @Test
+  public void updatesScheduledInterview() {
+    tester.create(scheduledInterview1);
+    Entity entity = datastore.prepare(new Query("ScheduledInterview")).asSingleEntity();
+    ScheduledInterview previousStoredScheduledInterview = tester.entityToScheduledInterview(entity);
+    ScheduledInterview updatedStoredScheduledInterview =
+        ScheduledInterview.create(
+            previousStoredScheduledInterview.id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+            "user@company.org",
+            "user3@mail.com");
+    tester.update(previousStoredScheduledInterview);
+
+    List<Boolean> actual = new ArrayList<Boolean>();
+    actual.add(previousStoredScheduledInterview.id() == updatedStoredScheduledInterview.id());
+    actual.add(
+        previousStoredScheduledInterview.when().equals(updatedStoredScheduledInterview.when()));
+    actual.add(
+        previousStoredScheduledInterview
+            .interviewerEmail()
+            .equals(updatedStoredScheduledInterview.interviewerEmail()));
+    actual.add(
+        previousStoredScheduledInterview
+            .intervieweeEmail()
+            .equals(updatedStoredScheduledInterview.intervieweeEmail()));
+
+    List<Boolean> expected = new ArrayList<Boolean>();
+    expected.add(true);
+    expected.add(false);
+    expected.add(true);
+    expected.add(false);
+
+    Assert.assertEquals(true, expected.equals(actual));
```

Comment:
No. You don't have to explicitly compare all components, that's what assertEquals is for.